pkg/loader/compose/v1v2.go

```diff
@@ -277,3 +322,80 @@ func checkLabelsPorts(noOfPort int, labels string, svcName string) error {
 	}
 	return nil
 }
+
+func findDuplicateVol(vols kobject.Volumes, volumes []kobject.Volumes) bool {
+	for _, v := range volumes {
+		if vols.MountPath == v.MountPath {
+			return true
+		}
+	}
+	return false
+}
+
+func handleVolume(svcName string, komposeObject kobject.KomposeObject) (volume []kobject.Volumes) {
+
+	if komposeObject.ServiceConfigs[svcName].VolumesFrom != nil {
```

Comment:
After certain time even the programmer forgets why s/he wrote a piece of code! :-D